xapian-core/cluster/clusterinternal.h

```diff
@@ -0,0 +1,120 @@
+/** @file clusterinternal.h
+ *  @brief Cluster API
+ */
+/* Copyright (C) 2017 Richhiey Thomas
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+
+#include <xapian/intrusive_ptr.h>
+
+#include <unordered_map>
+#include <vector>
+
+using namespace std;
```

Comment:
I'd probably avoid `using namespace std;` here as there's not much to qualify, but if you are going to use it, then please be consistent about `vector` vs `std::vector` below.